### PR TITLE
MSCO-16: Fix issue where the device family and model are incorrectly …

### PR DESCRIPTION
**Pull request by pcandia**
…imported

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -943,44 +943,41 @@ define(function(require) {
...
 						totalRecords: data.records.length,
 						columns: {
 							actual: data.columns.actual,
-							expected: expected
+							expected: expected,
+							others: []
 						},
-						recordsToReview: data.records.slice(0, 5),
+						recordsToReview: data.isDevices ? _.map(data.records, function(record) {
+							return _.chain(record)
```
Comment (joristirado) [inline app.js]: We don't want to mutate the original record here:
```suggestion
							return _.chain({})
```

File: app.js [hunk 2]
```diff
@@ -943,44 +943,41 @@ define(function(require) {
...
+								.merge(record, _.reduce(record, function(row, prop, index) {
```
Comment (joristirado) [inline app.js]: Isn't `record` an object?
```suggestion
								.merge(record, _.reduce(record, function(acc, value, prop) {
```

File: app.js [hunk 3]
```diff
@@ -943,44 +943,41 @@ define(function(require) {
...
+									if (modifierPerProp[index]) {
+										row[index] = modifierPerProp[index](prop);
+									}
+									return record;
+								}))
```
Comment (joristirado) [inline app.js]: Here we can abstract the if statement by using `lodash#identity`, which returns the first argument it receives:
```suggestion
									var modifier = _.get(modifiersPerProp, prop, _.identity);

									return _.set(acc, prop, modifier(value));
								}, {}))
```

Don't forget to give reduce an initial value (accumulator).

**Review by joristirado — COMMENTED** [review]
Some nitpicking on code structure, otherwise nicely done!

File: app.js (hunk 2 above)
Comment (pcandia) [reply app.js]: yeah, it is

**Review by joristirado — APPROVED** [review]
Looks good, merge away!

**Pull request merged**